src/plugins/collapsibleColumns/collapsibleColumns.js

```diff
@@ -90,6 +86,13 @@ class CollapsibleColumns extends BasePlugin {
    * @type {StateManager}
    */
   headerStateManager = null;
+  /**
+   * Map of hidden columns by the plugin.
+   *
+   * @private
+   * @type {null|HidingMap}
+   */
+  #hiddenColumnsMap = null;
```

Comment:
Interesting idea with keeping the map as private prop. Maybe we could refactor all plugins with maps to hide them? WDYT @wszymanski ?

Reply:
We can do it. As in the rest of the Handsontable. Although, we should remember that there is no way to get a registered map by its name. Maybe it should be changed? So far, we provided some examples of changing a map by accessing the private map.